IntroToCSharp/Pages/Activities/Booleans.razor

```diff
@@ -0,0 +1,111 @@
+@inherits CaptainCoder.Page
+@page "/activities/booleans"
+
+
+<h1>Booleans</h1>
+<Section Title="What are booleans?">
+    <p>
+        In C# a boolean or a “bool”, is a value that attempts to answer one of the simplest questions in C#.
+        Is this statement true or false? It is important to remember that booleans are working and used to answer that question!
+        In order to create a boolean we simple make a boolean value and, depending on the situation, set it to either true or false.
+    </p>
+    <p>
+        While this is a very simple data type it can be monumentally helpful when deciding how a program should proceed, 
+        make decisions or even use logic to solve a problem. Consider this problem below.
+    </p>
+
+    <MudContainer Align="align-center">
+    <MudIcon Icon="@(FlashlightSwitch ? Icons.Material.Filled.FlashlightOn : Icons.Material.Filled.FlashlightOff)" Title="@(FlashlightSwitch ? "Flashlight On" : "Flashlight Off")"/>
+    <MudSwitch @bind-Checked="@FlashlightSwitch" ThumbIcon="@(FlashlightSwitch==true ? Icons.Material.Filled.Done : Icons.Material.Filled.Close)" ThumbIconColor="@(FlashlightSwitch==true ? Color.Success : Color.Error)">IsLightSwitchOn: @FlashlightSwitch</MudSwitch>
+    <CodeBlock Filename="Booleans/IsLightSwitchOn.cs"/>
+    </MudContainer>
+
+    <p>
+        In this problem we can see that we have a boolean called IsLightSwitchOn. This is the question that our boolean is going to try to answer. For starters, you can see that right now the light switch is set to False. In other words the question “Is the Light Switch On?” is false, 
```

Comment:
Try and get used to using <MudText> instead of html p tags

Reply:
using???